src/wallet/wallet.cpp

```diff
@@ -4041,7 +4041,7 @@ std::shared_ptr<CWallet> CWallet::Create(interfaces::Chain& chain, const std::st
             }
 
             if (rescan_height != block_height) {
-                error = _("Prune: last wallet synchronisation goes beyond pruned data. You need to -reindex (download the whole blockchain again in case of pruned node)");
+                error = strprintf(_("Prune: last wallet synchronisation goes beyond pruned data. You need to %s (download the whole block chain again in case of pruned node)."), "-reindex");
```

Comment:
nit: blockchain vs block_chain

Reply:
https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md

Reply:
This isn't optech